src/Nethermind/Nethermind.Evm/EOF/EvmObjectFormat.cs

```diff
@@ -296,17 +298,8 @@ public bool ValidateBody(ReadOnlySpan<byte> container, in EofHeader header)
                 SectionHeader sectionHeader = header.CodeSections[sectionIdx];
                 (int codeSectionStartOffset, int codeSectionSize) = sectionHeader;
                 ReadOnlySpan<byte> code = container.Slice(codeSectionStartOffset, codeSectionSize);
-                if (!ValidateInstructions(code, header))
-                {
-                    if (Logger.IsTrace) Logger.Trace($"EIP-3670 : CodeSection {sectionIdx} contains invalid body");
```

Comment:
why remove this logs?

Reply:
they are redundant here, `ValidateInstructions` and `ValidateStackState` already log failure reason